src/app/query/bitcoin/stamps/stamps-by-address.query.ts

```diff
@@ -57,18 +57,18 @@ interface StampsByAddressQueryResponse {
   };
   data: {
     stamps: Stamp[];
-    src20: Src20[];
+    src20: Src20Token[];
   };
 }
 
 /**
  * @see https://stampchain.io/docs#/default/get_api_v2_balance__address_
  */
-async function fetchStampsByAddress(address: string): Promise<Stamp[]> {
+async function fetchStampsByAddress(address: string): Promise<StampsByAddressQueryResponse> {
   const resp = await axios.get<StampsByAddressQueryResponse>(
     `https://stampchain.io/api/v2/balance/${address}`
   );
-  return resp.data.data.stamps;
+  return resp.data;
```

Comment:
Maybe this can be `resp.data.data` here so that in the other places it's just `data.stamps` / `data.src20`?